### DISCUSSION TITLE
Shovel to move message from amqp to amqps queue

### DISCUSSION THREAD

**Discussion by krisnakp** [rabbitmq/rabbitmq-server#9048] [Questions]
Hi , 
 I am facing issue while creating the shovel instance to move the messages from rabbitmq-1's queue to rabbitmq-2's queue.
Current setup is rabbitmq-1 is running with tcp connect (5672), and rabbitmq-2 is running with ssl enable (5671).
I have create shovel with below properties, but the shovel status is in starting :
shovel-name : test
source : amqp://rmq1:rmqpass@rabbitmq1:5672/vh1
destination : amqps://rmq2:rmq2pass@rabbitmq2?cacertfile=/tls2/ca_certificate.pem&certfile=/tls2/client_certificate.pem&keyfile=/tls2/client_key.pem&verify=verify_peer.

can anyone help me , whether is there anything I am missing in configuring the shovel.

Thanks

**Comment by lukebakken** [comment]
Please provide the following _important_ information, without which we can't assist you:
* RabbitMQ and Erlang versions
* Definitions export from both nodes
* Full log files from both nodes

Do NOT paste the log files into your response. Instead, compress them and attach the compressed archive.
